### ISSUE TITLE
Footer als Tabelle sinnvoller?

### ISSUE THREAD

**Issue by moritzgloeckl** [sveneusewig/gnucash-german-taxinvoice#5]
Bei vielen Rechnung findet sich der Footer eher als Tabelle als mehrere Zeilen untereinander. Ich überlege ob es eventuell sinnvoller wäre den Footer als Tabelle darzustellen ähnlich wie:

<img width="455" alt="Screenshot 2021-01-21 at 00 05 33" src="https://user-images.githubusercontent.com/1828337/105251822-83308700-5b7c-11eb-8533-daeb6093dfc0.png">

**Comment by sveneusewig** [comment]
Eine gute Idee! Die Höhe des Footers muss per CSS so von der gesamten Blattgröße abgezogen werden, sonst wird immer eine neue leere Seite angehängt.
Für die Nutzer die andere Papierformate verwenden wollen ist das aktuell nicht möglich (außer man passt die german-taxinvoice.eguile.scm File das per Hand an). Hast du eine Idee wie man an innerhalb der GNUcash an das eingestellte Papierformat kommt?

**Comment by moritzgloeckl** [comment]
Ja das mit der Höhe vom Footer bzw mit der leeren Seite habe ich leider auch schon bemerkt. Vermutlich sollte der Footer eine fixe Höhe haben, das muss ich mir aber noch genauer ansehen.

Prinzipiell denke ich kann man das Papierformat über `gtk-paper-size` ([Doc](https://www.gnu.org/software/guile-gnome/docs/gtk/html/GtkPaperSize.html)) abrufen. Wie das aber konkret funktioniert kann ich dir aber leider auch noch nicht sagen! Fraglich bleibt für mich ob diese Funktion für ein German Taxinvoice Report notwendig ist bzw den Aufwand gerechtfertigt ist - man müsste dann vermutlich im CSS individuell auf jede Papiergröße eingehen. Ich traue mich aber zu sagen, dass vermutlich 99% der Nutzer das A4 Standardformat nutzen wird, da schlicht bei uns im DACH-Raum kaum US-Letter oder ähnliches verwendet wird. Einzig A5 oder Ähnliche könnte ich mir vorstellen. Bist du hier anderer Meinung?

**Comment by sveneusewig** [comment]
Richtig. Jedoch ist mir das immer noch zu statisch. Ich bin noch dabei an der Tabelle zu basteln. Die Ränder und das Papierformat versuche ich in das Eingebettete-CSS mit aufzunehmen.
